### ISSUE TITLE
[Accessibility][Windows & Mac desktop client] AddIn contextual control not accessible via keyboard

### ISSUE THREAD

**Issue by 3dot1four** [OfficeDev/office-js#606]
## Expected Behavior
We should be able to open AddIn context control using keyboard

## Current Behavior
Not able to open menu using the keyboard

## Steps to Reproduce, or Live Example
Insert an Addin and try accessing respective control using keyboard as shown in below image.

![image](https://user-images.githubusercontent.com/4102247/60879002-70868380-a25e-11e9-82e5-0ad6b1f7fc8a.png)

## Your Environment

* Platform [PC desktop, Mac, iOS, Office Online]: Desktop, Mac
* Host [Excel, Word, PowerPoint, etc.]: Excel
* Office version number: 11328.20368 (does not matter I guess)
* Operating System:  Windows and Mac

**Comment by grangeryy** [comment]
Bug 3480670 is created to track.

**Comment by magda-pearson** [comment]
This issue still exists within Outlook online Add-Ins. I tried to use One Note add-in and I wasn't able to go further than from X to the main frame (look at the screenshot). I used tab and arrows and no luck. I tested this in Edge 83.0.478.37 and Chrome 83.0.4103.61. This issue makes it impossible for keyboard users to access all Add-Ins' functionalities.
![2020-05-22_11h39_27](https://user-images.githubusercontent.com/65766373/82662115-354a3480-9c25-11ea-9ace-71eed390e215.png)

**Comment by grangeryy** [comment]
@lliu113  guess this is an add-in UX related improvement ask?

**Comment by lliu113** [comment]
@3dot1four - sorry for the very belated response to this. On windows, the shortcut to access the menu is Ctrl+Shift+F10. On Mac, it is accessibile using VoiceOver keys. 

@magda-pearson - sorry you're hitting this! It does appear to be a separate issue, so if you could open up another github issue that will help us track more effectively. 

I'm going to go ahead and close this one for now, feel free to reactivate if you have any additional questions.

**Comment by magda-pearson** [comment]
@lliu113 Thanks for your message. I'm not able to replicate it anymore, it occurs to be fixed. I can open the add-in via the keyboard in Chrome 90.0.4430.212.
